dev/tools/controllerbuilder/template/controller.go

```diff
@@ -297,10 +316,3 @@ func setStatus(u *unstructured.Unstructured, typedStatus any) error {
 
 	return nil
 }
-
-func ToOpenAPIDateTime(ts *timestamppb.Timestamp) *string {
```

Comment:
Mind explaining why removing this?

Reply:
This function has been moved to [maputils.go](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/2caba9831c3ba01b648504ca28e8c5d3fdcbfb99/pkg/controller/direct/maputils.go#L140)